modules/disaster-recovery/velero/README.md

```diff
@@ -0,0 +1,77 @@
+# Velero Module
+
+Velero is an open-source tool for backing up and restoring your Kubernetes cluster resources
+and persistent volumes. It provides disaster recovery, data migration, and storage migration solutions.
+
+[Here is the link to Velero official repository.](https://github.com/vmware-tanzu/velero)
+
+## What the base module contains
+
+- **CRDs:** Velero custom resource definitions
+- **Minio:**
+  - **Configs:** configurations of related resources (e.g. `BackupStorageLocation`, network
+  policies, Minio authentication, etc.)
+  - **Controller:** Minio deployment and service
+  - **RBAC:** Minio service account
+- **Resources (Velero):**
+  - **Configs:** configurations of related resources (namespace, network policies)
+  - **Controller:** base deployment for Velero
+  - **RBAC:** service account, `ClusterRole` and `ClusterRoleBinding`, with all the permissions needed to manage Velero resources
+
+## Flavors
+
+### GCP Storage
+
+- Includes settings and patches to use **GCP storage** instead of Minio.
+Follow the [instructions below](#gcp-storage-1) for a correct configuration.
+
+## Compatibility Matrix
+
+| Module Version | Tool Version   |
+|----------------|----------------|
+| 1.24.0         | 1.9.5	        |
```

Comment:
I don't know if there is a nice way to do it, but I think that is better to set all the versions of the tools here (velero, gcp plugin and minio) for completeness